src/nt2_fmt_plug.c

```diff
@@ -508,7 +508,7 @@ static void set_key_utf8(char *_key, int index)
 		keybuf_word += SIMD_COEF_32;
 	}
 
-	((unsigned int *)saved_key)[14*SIMD_COEF_32 + (index&3) + (index>>2)*16*SIMD_COEF_32] = len << 4;
+	((unsigned int *)saved_key)[14*SIMD_COEF_32 + (index&3) + (index>>SIMD_COEF32_BITS)*16*SIMD_COEF_32] = len << 4;
```

Comment:
I bet fixes like the ones above are needed in a LOT of formats. @jfoug already did many of them (in GETPOS macros and the like) but not all.